pkg/sync/adapter.go

```diff
@@ -197,6 +197,23 @@ func (a Adapter) pushLocal(path string, files []string, delFiles []string, isFor
 		}
 	}
 
+	if isForcePush {
+		// If isForcePush is set, then the entire local directory contents will be copied over by CopyFile(...), thus
+		// we first need to clear the remote target folder.
+
+		err = exec.ExecuteCommand(a.Client, compInfo, []string{"rm", "-rf", syncFolder}, true, nil, nil)
```

Comment:
maybe rm and mkdir in one call?

Reply:
Hi @mik-dass, they are intentionally kept separate as the `rm -rf` may fail (which is fine), but the `mkdir` should not fail. 

I've updated that logic in the function since you reviewed it, take a look and see what you think. It is now:
1) Do `rm -rf (sync folder)` to clear out the directory (log any errors, but continue to step 2)
2) Do `mkdir -p (sync folder)` to ensure the directory exists (this should never fail, so err is fatal to this function)
3) Verify that the `(sync folder)` exists, and contains no files: this checks that both step 1 and step 2 worked as expected.